src/config/hpc/hpc.ipxe

```diff
@@ -43,6 +40,29 @@ set server-ip ${dhcp-server}
 set script-url http://${dhcp-server}
 # If the hostname is set in the DHCPOFFER use that for a personalized script.
 isset ${hostname} && set script-url ${script-url}/${hostname}/script.ipxe || set script-url ${script-url}/boot/script.ipxe
+
+# Get cpu architecture
+iseq ${buildarch} arm64 && goto arm64 ||
```

Comment:
This variable seems new, is this actually set by iPXE behind the scenes or where does `${buildarch}` come from?